Java/Endpoint Examples/Multipart Payload/PDFWithRedactedTextPreview.java

```diff
@@ -0,0 +1,62 @@
+import io.github.cdimascio.dotenv.Dotenv;
+import java.io.File;
+import java.io.IOException;
+import java.util.concurrent.TimeUnit;
+import okhttp3.*;
+import org.json.JSONObject;
+
+public class PDFWithRedactedTextPreview {
+
+  // Specify the path to your file here, or as the first argument when running the program.
+  private static final String DEFAULT_FILE_PATH = "/path/to/file";
+
+  // Specify your API key here, or in the environment variable PDFREST_API_KEY.
+  // You can also put the environment variable in a .env file.
+  private static final String DEFAULT_API_KEY = "xxxxxxxx-xxxx-xxxx-xxxx-xxxxxxxxxxxx";
+
+  public static void main(String[] args) {
+    File inputFile;
+    if (args.length > 0) {
+      inputFile = new File(args[0]);
+    } else {
+      inputFile = new File(DEFAULT_FILE_PATH);
+    }
+
+    final Dotenv dotenv = Dotenv.configure().ignoreIfMalformed().ignoreIfMissing().load();
+
+    final String redaction_options =
+        "[{\"type\":\"preset\",\"value\":\"uuid\"},{\"type\":\"regex\",\"value\":\"(\\\\+\\\\d{1,2}\\\\s)?\\\\(?\\\\d{3}\\\\)?[\\\\s.-]\\\\d{3}[\\\\s.-]\\\\d{4}\"},{\"type\":\"literal\",\"value\":\"word\"}]";
```

Comment:
```suggestion
        "[{\"type\":\"preset\",\"value\":\"email\"},{\"type\":\"regex\",\"value\":\"(\\\\+\\\\d{1,2}\\\\s)?\\\\(?\\\\d{3}\\\\)?[\\\\s.-]\\\\d{3}[\\\\s.-]\\\\d{4}\"},{\"type\":\"literal\",\"value\":\"word\"}]";
```

I get this error for this set of options. Replacing with `email` (or some other valid option) worked.

```
{"error": "uuid is not a valid preset redaction option. See documentation for valid options."}
```